Lesson27/aysel/create.js

```diff
@@ -0,0 +1,29 @@
+document.getElementById('create-post-form').addEventListener('submit', function(event) {
+    event.preventDefault();
+    createPost();
+});
+
+function createPost() {
+    const title = document.getElementById('title').value;
+    const body = document.getElementById('body').value;
+
+    const newPost = {
+        title: title,
+        body: body,
+        userId: 1
```

Comment:
Same here are in the `update.js` file, don't set userId, only update title and body